server/apps/main/templates/main/home.html

```diff
@@ -51,10 +51,15 @@ <h2 class="section-heading">About AutSPACEs</h2>
     </div>
   </div>
   <div class="row">
-    <div class="col float-right">
-      <a href="{% url 'main:about_us' %}">
-        <button class="btn btn-outline-primary btn-home-about btn-lg float-right" type="button">Read More</button>
-      </a>
+    <div class="col">
+      <div class="float-right">
+        <a href="{% url 'main:share_exp' %}" class="mr-3">
+          <button class="btn btn-outline-primary btn-home-about btn-lg mb-3" type="button">Add Story</button>
+        </a>
+        <a href="{% url 'main:about_us' %}" class="mt-3">
+          <button class="btn btn-outline-primary btn-home-about btn-lg mb-3" type="button">Read More</button>
+        </a>
```

Comment:
I think those button declarations are duplicating the links if defined like this! The one below should work: 

```suggestion
<a href="{% url 'main:share_exp' %}" class="btn btn-outline-primary btn-home-about btn-lg mb-3">
          Add Story
        </a>
        <a href="{% url 'main:about_us' %}" class="btn btn-outline-primary btn-home-about btn-lg mb-3">
          Read More
        </a>
```

Reply:
oh, and relatedly: currently when logged out the "add story" button always redirects to the front page, due to the redirect being set to that instead of the register page

Reply:
Yeah, I think every time I noticed that double declaration I tried fixing it, but I'm sure there's some places left! 😂

Reply:
They should all be switchd now; at least the ones I could find.